lib/iris/plot.py

```diff
@@ -1036,11 +1092,16 @@ def citation(text, figure=None):
         Target :class:`matplotlib.figure.Figure` instance. Defaults
         to the current figure if none provided.
 
+    * axes: :class:`matplotlib.axes.Axes` instance to call the
+         contour function on. Defaults to the current axes if none
+         provided.
+
     """
 
     if text is not None and len(text):
-        if figure is None:
+        if figure is None and not axes:
             figure = plt.gcf()
         anchor = AnchoredText(text, prop=dict(size=6), frameon=True, loc=4)
         anchor.patch.set_boxstyle('round, pad=0, rounding_size=0.2')
-        figure.gca().add_artist(anchor)
+        axes = axes if axes else figure.gca()
+        axes.add_artist(anchor)
```

Comment:
I don't think this needs to be axes - it could just as well be ```figure.patches.append(anchor)``` (untested, but along those lines). If that is the case, then axes is simply not needed as a kwarg.